### PR TITLE
Fix #136: Update tests to adapt changes in document upload

### LINKED ISSUE #136: Update tests to adapt changes in document upload
Update tests according the changes at https://github.com/wultra/enrollment-server/issues/512

### PR DESCRIPTION
**Pull request by banterCZ**
Depends on https://github.com/wultra/enrollment-server/pull/515

### CODE REVIEW

File: powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java [hunk 1]
```diff
@@ -205,7 +205,7 @@ private void processDocuments(final TestContext context) throws Exception {
         }
 
         assertIdentityVerificationStateWithRetries(
-                new IdentityVerificationState(IdentityVerificationPhase.DOCUMENT_VERIFICATION, IdentityVerificationStatus.IN_PROGRESS));
+                new IdentityVerificationState(IdentityVerificationPhase.DOCUMENT_UPLOAD, IdentityVerificationStatus.IN_PROGRESS));
```
Comment (banterCZ) [inline powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java:208]: @kober32 Here you can see that after upload of the first document, you should expect `DOCUMENT_UPLOAD / IN_PROGRESS`

**Comment by romanstrobl** [comment]
I am concerned that the tests still have:

```
powerauth.test.identity.verificationOnSubmitEnabled=true
```

Although this logic is currently disabled on server.

Are we testing the same thing?

**Review by romanstrobl — APPROVED** [review]
I am happy to see minimal changes in tests.

I am a bit concerned about relevance of tests since the server runs now with `verificationOnSubmitEnabled=false`, while the tests run with `verificationOnSubmitEnabled=true`. However, this could be another ticket.

**Review by romanstrobl — CHANGES_REQUESTED** [review]
I ran the tests and there are some failures. Postponing approval after tests are passing.

**Comment by banterCZ** [comment]
Tests passed

**Review by romanstrobl — APPROVED** [review]
Happy to see expected changes and that these changes are minimal.

**Comment by romanstrobl** [comment]
However, I still see: `verificationOnSubmitEnabled=true` in this branch, which does not correspond to backend configuration. Can we change the default value?

**Review by romanstrobl — APPROVED** [review]
Approved.

**Pull request merged**